web-client/src/types/openapi-generated.ts

```diff
@@ -303,6 +313,16 @@ export interface components {
       checkerId: number;
       validatorId?: number;
     };
+    UserError: {
+      id?: string;
+      raw?: { [key: string]: unknown };
```

Comment:
![download](https://user-images.githubusercontent.com/3090032/185787355-65dddd0c-8856-4f2d-9d1e-ef7ce4655d91.jpeg)

Hehehe. You should probably recompile the openapi thing!

Reply:
omg woops hAHAHAHAH